### PR TITLE
Docker configuration files for peering-manager

### PR DESCRIPTION
**Pull request by zimage**
### Fixes:
I have added scripts to create docker containers for the database and the web app.

```
build-host:peering-manager devops$ docker-compose build --build-arg 
NAPALM_PASSWORD='secret' --build-arg NAPALM_USERNAME=ix-manager
...
...
build-host:peering-manager devops$ docker-compose up
```
The app is then available at http://0.0.0.0:8000 with username:admin and password: password

### CODE REVIEW

**Comment by zimage** [comment]
Note: The database's data is not yet stored in local storage. If the db container is rebuilt, then the app will lose all its data. This docker feature is currently useful mostly for quick deployment for evaluation. I will be adding persistent storage for the database soon.

File: docker/configuration.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This is a list of valid fully-qualified domain names (FQDNs) for this server.
+# The server will not permit write access to the server via any other
+# hostnames. The first FQDN in the list will be treated as the preferred name.
+#
+# Example: ALLOWED_HOSTS = ['peering.example.com', 'peering.internal.local']
+import os
+ALLOWED_HOSTS = ["*"]
+
+# Must be unique to each setup (CHANGE IT!).
+SECRET_KEY = "ef7npku*djrj_r4jt4cojo8^j@2($$@05e(eq_mn!ywx*jg0vy"
```
Comment (2bithacker) [inline docker/configuration.py]: Should probably have this passed in via an ENV var

Comment (philodavies) [reply docker/configuration.py]: I haven't seen any traction on this PR. If you'd like I can take over and make the edit to get the secret from an ENV var.

**Comment by mxhob1** [comment]
I'm also happy to pick this up to add the ENV variable and persistent storage so that it can be merged into master.

**Comment by respawner** [comment]
I'm closing this PR as there is no progress on it. Please open an issue and link a PR to it if you're willing to work on this.

**Pull request closed without merging**